apps/vr-tests-react-components/src/stories/Spinner.stories.tsx

```diff
@@ -8,7 +8,7 @@ import { TestWrapperDecoratorNoAnimation } from '../utilities/TestWrapperDecorat
 // or photo. This wrapper ensures a dark background so the Spinners
 // are consistently visible.
 const InvertedWrapper: React.FC = ({ children }) => {
-  return <div style={{ background: tokens.colorPaletteBerryForeground1, padding: '10px' }}>{children}</div>;
+  return <div style={{ background: tokens.colorBrandBackgroundStatic, padding: '10px' }}>{children}</div>;
```

Comment:
What do you think about using the color from #24494 for this?

Reply:
`colorNeutralBackgroundInverted` which is used in that PR will not work for spinner.
According to the design spec and your comment in spinner vr stories `<Spinner appearance="inverted">` is to be used on top of dark background regardless the theme:
<img width="844" alt="image" src="https://user-images.githubusercontent.com/9615899/186604674-7a937c75-66e2-45bb-bb6b-d4e3ef0a9197.png">
That means that even in Dark theme the spinner is white. But `colorNeutralBackgroundInverted` is white in the Dark theme - check the [screener for **Spinner converged: Primary with Size Tiny - Dark Mode** in #24494](https://screener.io/v2/states/4V5IW.microsoft-fluentui-react-components/fix%2Fcolor-token-for-appearance-backgrounds/1024x768/Chrome/spinner-converged-primary-with-size-tiny-dark-mode-0) - that renders white on white.